science/nektarpp/Portfile

```diff
@@ -7,12 +7,12 @@ PortGroup               gitlab 1.0
 PortGroup               boost 1.0
 
 gitlab.instance         https://gitlab.nektar.info
-gitlab.setup            nektar nektar 5.2.0 v
+gitlab.setup            nektar nektar 5.3.0 v
 
-boost.version           1.71
+boost.version           1.76
 
 name                    nektarpp
-revision                2
+revision                0
 
 categories              science
 platforms               darwin
```

Comment:
this is the default and can thus be removed

Reply:
Have removed (and also removed comments around 1.71)

Reply:
I was talking about the `platforms` field, that's the default and should be removed. Please leave the `revision 0` line